assets/scss/books.scss

```
@@ -620,10 +944,55 @@ $book-accent: #478079;
+// inherit the backdrop above — it's given its own light surface here instead
+// (via Bootstrap's own `--bs-modal-*` custom properties), matching the page's
+// light theme, so text/borders need the same dark-on-light tokens used
+// everywhere else and the shared gold accent needs the deeper
+// $book-accent-fg to stay AA-contrast.
 // ---------------------------------------------------------------------------
 
 .book-detail-modal {
+  .modal-content {
+    --bs-modal-bg: #{$cinema-bg-2};
+    --bs-modal-color: #{$cinema-fg};
+    --bs-modal-border-color: rgba($cinema-fg, 0.1);
```

Stylelint (17.14.0)

[error] Expected variable $cinema-fg to be interpolated when using it with --bs-modal-border-color (scss/dollar-variable-no-missing-interpolation)

(scss/dollar-variable-no-missing-interpolation)